garage/tf/policies/base2.py

```diff
@@ -103,6 +103,32 @@ def get_global_vars(self):
         """Get global vars."""
         return self._variable_scope.global_variables()
 
+    def add_model(self, model):
+        self._models.append(model)
+
+    def build_models(self, input_var, name=None):
+        out = input_var
+        for model in self._models[:-1]:
+            out = model.build(out, name=name)
+        self.model = self._models[-1]
```

Comment:
could you remind me why the last model is `self.model`?

Reply:
This is actually a temp-fix from my previous implementation. We don't need `self.model` anymore, as they now become `models`. Nice catch.